.github/workflows/build.yml

```diff
@@ -347,6 +347,61 @@ jobs:
       - name: Ensure benchmarking compiles
         run: cargo check --release --features=runtime-benchmarks
 
+  build-rt-binary:
```

Comment:
What is the purpose of this one, it looks like a copy of build ?

Reply:
It is a copy of build, the purpose being so that we can get a node binary of the gitTag associated with a released runtime. With this we can generate a plain spec file, with working genesis blob, so that we can simulate the situation of a locally built client node, running an old RT, upgrading to a locally built runtime

Reply:
Shouldn't we use the previously released RT instead ?

Reply:
the released runtime doesn't have a node binary associated with it, it's just a wasm blob. We need the node binary of that branch, so that we can generate a plain spec associated with that RT code.

Otherwise we get into weird situations whereby the genesis is wrong

Reply:
You can retrieve it from docker, like we do for polkadot
released runtime should have a moonbeam docker image matching their sha

Reply:
DONE